dev-docs/source/ReleaseChecklist.rst

```diff
@@ -61,12 +61,12 @@ Monday then the names of the days will have to be changed.
 | | **Wednesday**,                |                           | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Manager <release-manager-checklist>`                     |
 | | 1 week & 3 days               |                           | Maintenance Tasks & Next release development  |                                                                          |
 +---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
-| | **Tuesday**,                  | Beta Testing Ends         | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Quality Assurance Manager <quality-assurance-manager-checklist>` |
-| | 4 days                        |                           | Maintenance Tasks & Next release development  | | :ref:`Release Editor <release-editor-checklist>`                       |
-|                                 |                           |                                               | | :ref:`Release Manager <release-manager-checklist>`                     |
+| | **Tuesday**,                  | Beta Testing Ends         | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Editor <release-editor-checklist>`                       |
+| | 4 days                        |                           | Maintenance Tasks & Next release development  | | :ref:`Release Manager <release-manager-checklist>`                     |
 +---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
-| | **Friday**,                   | Release Eve               | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Manager <release-manager-checklist>`                     |
-| | 1 day                         |                           | Maintenance Tasks & Next release development  |                                                                          |
+| | **Friday**,                   | | Release Eve             | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Quality Assurance Manager <quality-assurance-manager-checklist>` |
```

Comment:
I would argue these last two rows the task priorities should shift to:
`Packaging, Blocker bug fixes, Testing`


`Blocker bug fixes, Testing, Release Announcements.`

As we tend to pause maintenance / development to test our final installers and next development cycle is out of scope for this (as it's a reference for the various release manager roles)